src/mame/drivers/gunpey.cpp

```diff
@@ -235,16 +235,22 @@ struct huffman_node_s
 };
 
 
+constexpr u32 get_vmem_addr(u16 x, u16 y)
```

Comment:
`vmem` is ambiguous (virtual memory?), should really be `vram` like everything else in the state machine.

Reply:
vmem means video memory, both video ROM and RAM.
Both has same addressing format(2048x2048 size framebuffer-like addressing).

Reply:
So be `get_video_addr[ess]`.